api/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Can we add an example that leverages one of these settings?

Reply:
Yes, we can add a mandarine framework example since Mandarine is the framework that needs that tsconfig.